dist/index.js

```diff
@@ -0,0 +1,96 @@
+/*
```

Comment:
ayayayaya 🔴 

remove from commit
that happened cause we haven't good structure of our project (next step) and `.gitignore` can't recognise what it can or can't commit